### PR TITLE
Adds home screen layout

### PR DESCRIPTION
**Pull request by DanialBetres**
Here's the Initial home screen view - but I have some questions/comments/concerns 

- For some reason storybook is not applying any css? 
- How do you feel dropping storybook lol. Only reason is just because i'm in favour of quicker development. I'll still be coding like as if we were using storybook so separating logic from components and things like that.
- We still need to add the font family. Are we sticking with the font family in the mocks or is there another font family you wanna use? 
- Are we developing for a mobile screen or web or both? If it's both then how would we handle the icon size for example for the home screen? We can make everything else look bigger but i'm not sure what to do with the icons.
- I also didn't add typescript yet... I don't know it yet and I was too lazy to learn it now - I'll add it later.

### CODE REVIEW

File: src/HomeScreen.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+
```
Comment (donghee214) [inline src/HomeScreen.js]: Add interface here to define what type the props will be

File: src/assets/images/pencil-icon.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="49" height="49" viewBox="0 0 49 49" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (donghee214) [inline src/assets/images/pencil-icon.svg]: What do you feel about wrapping all of our svgs in a react component? That way we can drive props into the svgs and change/animate changing colours & size

File: src/HomeScreen.js (hunk 1 above)
Comment (DanialBetres) [reply src/HomeScreen.js]: still need to learn typescript :( 😢

File: src/assets/images/pencil-icon.svg (hunk 2 above)
Comment (DanialBetres) [reply src/assets/images/pencil-icon.svg]: hmm yeah we can do that 👍

**Pull request merged**